doc/manual/source/interaction/web-api.rst

```diff
@@ -284,7 +284,7 @@ Porcelain commands supported
 
 	   Returns the name of the remote, the branch name before and after, and the value before and after.
 
- .. note:: If remote is not specified it will try to fetch from a remote named origin.
+  .. note:: If remote is not specified it will try to fetch from a remote named origin.
```

Comment:
The Note box isn't quite spaced enough. It's better than it was, but still not lined up like the fix you did above for `Checkout`. I think it needs another space/tab or 2.
Current:
![current2](https://cloud.githubusercontent.com/assets/1253074/25404015/fb82ca36-29c3-11e7-826b-85b6db8b48c5.png)
Edit:
![edit2](https://cloud.githubusercontent.com/assets/1253074/25404028/01fcfeb8-29c4-11e7-9589-09c047f04248.png)